assets/section-main-product.css

```diff
@@ -231,11 +231,17 @@ fieldset.product-form__input .form__label {
   background-color: rgba(var(--color-background), 0.3);
 }
 
-.product-form__input input[type='radio']:disabled + label {
+.product-form__input input[type='radio']:disabled + label,
+.product-form__input input[type='radio'].disabled + label {
   border-color: rgba(var(--color-foreground), 0.1);
   color: rgba(var(--color-foreground), 0.4);
```

Comment:
```suggestion
  color: rgba(var(--color-foreground), 0.6);
```

I think we can bump up the opacity to be at `0.6` so that we're safer in terms of color contrast. Right now when it's disabled, the contrast is too low to be considered accessible: 
![](https://screenshot.click/15-21-loun2-f438c.png)